### DISCUSSION TITLE
Could you give me an example how to deploy micro frontend ?

### DISCUSSION THREAD

**Discussion by hoanghd218** [web-infra-dev/modern.js#4718] [Q&A]
Could you give me an example how to deploy micro frontend ?

**Comment by chenjiahan** [comment]
Which platform do you need to deploy your MFE application to?

**Reply by hoanghd218** [reply]
Hi @chenjiahan , I want to deploy it on AWS. Could you please give me some suggestions? Thank you.

**Reply by chenjiahan** [reply]
We currently do not have a deployment guide for AWS. You can refer to ["output files"](https://modernjs.dev/builder/en/guide/basic/output-files.html) to understand the dist directory structure of Modern.js and how to custom the output structure.

For MFE, the deployment becomes more complex, and the deployment forms of different MFE frameworks may be different. We will provide a deployment guide in the next quarter.

**Reply by hoanghd218** [reply]
Thank you. Wait to see it.
